game.ts

```diff
@@ -0,0 +1,29 @@
+import * as readline from "node:readline";
+import { stdin as input, stdout as output } from "node:process";
+
+//createInterface is a callable of Node's readLine module (see documentation)
+//In this way, we can connect stdin and stdout
+const rl = readline.createInterface({ input, output });
+
+function generateComputerMove() {
+  return String(Math.round(Math.random() * 2));
+}
+
+export function play(question: string) {
+  let computerMove = generateComputerMove();
+  rl.question(question, (userMove) => {
+    rl.write(`You choosed:  ${userMove}\nComputer choosed:  ${computerMove}\n`);
+    if (userMove == computerMove) {
```

Comment:
```suggestion
    if (userMove === computerMove) {
```
It's best to always use `===` for safety.

Reply:
done